sys/kern/kern_flag.c

```diff
@@ -105,10 +106,10 @@ kern_flag_captured(struct thread *td, const char * __capability message,
 	}
 
 	log(LOG_ALERT,
-	    "pid %d (%s), jid %d, uid %d: captured flag source: (%s) "
+	    "pid %d (%s), ABI: %s, jid %d, uid %d: captured flag source: (%s) "
 	    "message: (%s) key: %s\n", p->p_pid, p->p_comm,
-	    p->p_ucred->cr_prison->pr_id, td->td_ucred->cr_uid, src_buf,
-	    msg_buf, key_buf);
+	    p->p_sysent->sv_name, p->p_ucred->cr_prison->pr_id,
```

Comment:
One thing I wasn't sure of is if p_sysent can be NULL?

Reply:
Looks like no, `proc0` gets `null_sysvec`

Reply:
Ah, that's interesting.  Normally we use this from system call context when it is never NULL, hadn't considered kproc's.